src/modules/auth/auth.service.ts

```diff
@@ -152,32 +162,33 @@ export default class AuthenticationService {
       const isMatch = await bcrypt.compare(password, user.password);
 
       if (!isMatch) {
-        throw new UnauthorizedException({
+        return {
           status_code: HttpStatus.UNAUTHORIZED,
           message: INVALID_CREDENTIALS,
-        });
+        };
       }
 
-      const access_token = this.jwtService.sign({ id: user.id });
+      const access_token = this.jwtService.sign({ id: user.id, sub: user.id });
 
       const responsePayload = {
         access_token,
         data: {
           user: {
+            id: user.id,
             first_name: user.first_name,
             last_name: user.last_name,
             email: user.email,
-            id: user.id,
           },
         },
       };
 
-      return { message: 'Login successful', ...responsePayload };
+      return { message: LOGIN_SUCCESSFUL, ...responsePayload };
     } catch (error) {
```

Comment:
we can do without this try/catch blocks

Reply:
we can but it using it here serves a purpose